src/lib.rs

```diff
@@ -96,7 +96,7 @@
 //! [COSE]: https://tools.ietf.org/html/rfc8152
 //! [CBOR]: https://tools.ietf.org/html/rfc7049
 
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
 #![deny(rustdoc::broken_intra_doc_links)]
 extern crate alloc;
```

Comment:
Should the inclusion of this create be guarded by `#[cfg(not(feature = "std"))]`?

Reply:
For a `std` build I think we still need this so that the contents of `alloc` are available under the `alloc` name.   (In other words: if we did have a conditional here, we'd also have to change all of the `use alloc::something` imports too.)